tests/integrations/raftstore/test_lease_read.rs

```diff
@@ -403,3 +403,32 @@ fn test_read_index_when_transfer_leader() {
     );
     drop(cluster);
 }
+
+#[test]
+fn test_local_read_cache() {
+    let mut cluster = new_node_cluster(0, 3);
+    configure_for_lease_read(&mut cluster, Some(50), None);
```

Comment:
50ms is too short for this case.

Reply:
Why?

Reply:
How about if the test blocked by system load is high for several milliseconds?